Automattic/wc-calypso-bridge
# Change store on wpcom order link to /wp-admin/post.php

Fixes #649 


This PR removes `woocommerce_get_edit_order_url` filter, which changes edit order URL to WPCOM store link. The order link is now linked to `:store-url/wp-admin/post.php?post=:order-id&action=edit`

This change is needed as we are removing the store from wordpress.com.

### Test Instruction

1. Create a new atomic site at https://mc.a8c.com/atomic/ephemeral-sites/
2. Do not check `wpcomsh` 
3. Checkout this branch and zip it or use the attached zip file.
[wc-calypso-bridge.zip](https://github.com/Automattic/wc-calypso-bridge/files/6025700/wc-calypso-bridge.zip)


4. Upload the zip file to your atomic site and activate it
5. Add a new product
6. Open a new incognito browser window and place a new order
7. Check your email and confirm the link

#### Screenshots

![Screen Shot 2021-02-18 at 10 09 34 PM copy](https://user-images.githubusercontent.com/4723145/108465065-2802ba80-7236-11eb-8ae2-6b24ffa27034.jpg)
